.pre-commit-config.yaml

```
@@ -15,13 +15,13 @@ repos:
       - id: trailing-whitespace
 
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.216
+    rev: v0.0.223
     hooks:
       - id: ruff
         args:
           - --fix
 
   - repo: https://github.com/psf/black
-    rev: 23.1a1
+    rev: 22.12.0
```

We're using pre-release version of black

```suggestion
    rev: 23.1a1
```